src/main/java/mate/academy/spring/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.spring.model.dto.response;
+
+import mate.academy.spring.validation.Email;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseDto {
+    @Email
+    private String email;
```

Comment:
```suggestion
    private String email;
```
no need to validate response